lib/omniauth/strategies/redbooth.rb

```diff
@@ -2,62 +2,34 @@
 
 module OmniAuth
   module Strategies
+    # @see https://redbooth.com/api/api-docs/
     class Redbooth < OmniAuth::Strategies::OAuth2
-      # Give your strategy a name.
-      option :name, 'redbooth'
+      REDBOOTH_URL = 'https://redbooth.com'
 
+      option :name, 'redbooth'
```

Comment:
I don't know where or how omniauth is going to use it, I'd rather not freeze something that might be used somewhere in a way which breaks something.